src/VideoPlayer/index.js

```diff
@@ -60,6 +61,16 @@ const state = {
   skipTime: false,
   playAfterSeek: null,
 }
+const subtitles = {
+  hasSubtitles: false,
+  currentSubtitle: '',
+  previousSubtitle: '',
+  clear: () => {
+    this.hasSubtitles = false
+    this.currentSubtitle = ''
+    this.previousSubtitle = ''
+  },
```

Comment:
I don't think this works as expected, since `this` is not referring to any class instance. Because this is not a class Component, I would declare it as a separate function `clearSubtitles` or `resetSubtitles` to revert to the default values

Reply:
Updated